plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/BaseElasticsearchConnectorTest.java

```diff
@@ -955,6 +956,44 @@ public void testMixedArray()
                 "VALUES NULL, ARRAY['hello'], ARRAY['foo', 'bar']");
     }
 
+    @Test(dataProvider = "emptyNumericFieldProvider")
+    public void testEmptyNumericField(String typeDefinition)
+            throws IOException
+    {
+        String indexName = "emptynumeric_" + randomTableSuffix();
+
+        @Language("JSON")
+        String mapping = "" +
+                "{" +
+                "  \"properties\": { " +
+                "    \"numeric_column\": " + typeDefinition +
+                "  }" +
+                "}";
+
+        createIndex(indexName, mapping);
+        index(indexName, ImmutableMap.of("numeric_column", ""));
+
+        assertQuery(
+                "SELECT numeric_column FROM " + indexName,
+                "VALUES NULL");
+
+        deleteIndex(indexName);
+    }
+
+    @DataProvider
+    public Object[][] emptyNumericFieldProvider()
```

Comment:
You can test all of these in a single query instead of using a provider and iterating over each type independently. It will make the test easier to read and debug if there are issues.

Reply:
Updated.